frontend/src/pages/SessionForm/pages/participants.js

```diff
@@ -21,57 +22,75 @@ import FormItem from '../../../components/FormItem';
 import useTrainingReportRole from '../../../hooks/useTrainingReportRole';
 import useTrainingReportTemplateDeterminator from '../../../hooks/useTrainingReportTemplateDeterminator';
 import UserContext from '../../../UserContext';
-import PocCompleteView from '../../../components/PocCompleteView';
-import ReadOnlyField from '../../../components/ReadOnlyField';
 import RecipientsWithGroups from '../../../components/RecipientsWithGroups';
+import ParticipantsReadOnly from '../components/ParticipantsReadOnly';
 
 const placeholderText = '- Select -';
 
+const ROLE_OPTIONS = [
```

Comment:
Not sure if we care to stick these in common.

Reply:
Only if we are going to use it somewhere else, I think